src/OnboardingSPA/steps/SiteGen/SiteDetails/index.js

```diff
@@ -146,6 +156,12 @@ const SiteGenSiteDetails = () => {
 							}
 							customChildren={ true }
 						>
+							<LanguageSelection
+								labgeageSelectionLabel={ content.languageSelectionLabel }
+								languageList={ content.languageList }
+								selectedLocale={ selectedLocale }
+								setSelectedLocale={ setSelectedLocale }
+							/>
```

Comment:
to prevent confusion and lessen needed users decisions, can we always default this to 'English/US`?

Meaning the select field should always have a value and 'English/US' is default.

Reply:
Defaulting to English/US code added.